src/codeu/chat/database/Schema.java

```diff
@@ -4,9 +4,13 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import codeu.chat.util.Logger;
+
 public abstract class Schema {
 
-  Map<String, String> fields;
+  private final static Logger.Log LOG = Logger.newLog(Schema.class);
```

Comment:
nit: `private static final` is the usual way to write this, mostly just for readability and consistent style.